services/basic_types.proto

```diff
@@ -79,9 +79,16 @@ message AccountID {
     int64 realmNum = 2;
 
     /**
-     * A nonnegative account number unique within its realm
-     */
-    int64 accountNum = 3;
```

Comment:
Please don't remove fields from protobufs; only deprecate them.

Reply:
It's not removed, it's moved to a oneof

Reply:
I assumed moving a field into a `oneof` would break parsing code, but it seems it doesn't.